portal/v2/src/ClusterDetailListComponents/ClusterOperatorsWrapper.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState, useEffect, useRef } from "react"
+import { AxiosResponse } from 'axios';
+import { FetchClusterOperators } from '../Request';
+import { ICluster } from "../App"
+import { IMessageBarStyles, MessageBar, MessageBarType, Stack } from '@fluentui/react';
+import { clusterOperatorsKey } from "../ClusterDetail";
+import { ClusterOperatorListComponent } from "./ClusterOperatorList";
+
+export interface ICondition {
+  status: string,
+  reason: string,
+  lastUpdated: string,
+  message: string
+}
+
+export interface IClusterOperator {
+  name: string,
+  available: string,
+  conditions?: ICondition[],
+}
+
+export function ClusterOperatorsWrapper(props: {
+  currentCluster: ICluster
+  detailPanelSelected: string
+  loaded: boolean
+}) {
+  const [data, setData] = useState<any>([])
```

Comment:
Two things here, I think we should use a more descriptive name than `data` and I don't think we should use `any` here as we control what data we set.

Reply:
Fixed